api-schema/src/main/java/org/stellar/anchor/api/sep/AssetInfo.java

```diff
@@ -15,10 +15,17 @@ public class AssetInfo {
   String issuer;
 
   public String getAssetName() {
-    if (issuer != null) {
-      return schema + ":" + code + ":" + issuer;
+    return schema + ":" + makeAssetName(code, issuer);
+  }
+
+  public static String makeAssetName(String assetCode, String assetIssuer) {
```

Comment:
By reading the names, the purposes of these functions seem unclear to me. Can we merge or comment?

Reply:
I'll update the method names and add Javadoc.